ui/Screen/Bsod.svelte

```diff
@@ -27,7 +27,8 @@
   };
 
   const support = () => {
-    window.location.href = "https://matrix.to/#/#support:radicle.community";
+    window.location.href =
+      "https://matrix.radicle.community/#/room/#support:radicle.community";
```

Comment:
Should we account for the potential rename to `radicle:support`?

Reply:
For the `general` channel, only the name was changed, but not the address. The link is still https://matrix.radicle.community/#/room/#general:radicle.community
So if we do the same for the support channel, the link would not have to be changed.
<img width="604" alt="Screenshot 2021-01-08 at 10 59 00" src="https://user-images.githubusercontent.com/16262137/104001840-8604a280-51a0-11eb-8cea-1e6d18455eaf.png">